tests/boardfarm_plugins/boardfarm_sample_plugin/tests/initial_ap_config.py

```diff
@@ -0,0 +1,42 @@
+
+import boardfarm
+
+from boardfarm.tests import bft_base_test
+
+
+class InitialApConfig(bft_base_test.BftBaseTest):
+    '''
+    PrplMesh sample test case, no actual testing, it just passes
+    '''
```

Comment:
Docstring coding style:
```suggestion
    '''prplMesh sample test case, no actual testing, it just passes.'''
```